src/PublicShareSidebarTrigger.vue

```diff
@@ -0,0 +1,69 @@
+<!--
+  - @copyright Copyright (c) 2022 Daniel Calviño Sánchez <danxuliu@gmail.com>
+  -
+  - @license GNU AGPL version 3 or any later version
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+  -
+  -->
+
+<template>
+	<Button @click="$emit('click')">
```

Comment:
Needs aria-label attribute as per console warning

Reply:
I asked Marco about the aria label and then forgot to add it :facepalm: 

Anyway, I have added it but I can not get rid of the console warning. I guess I made some silly mistake when passing the property to the button or something like that, but I can not find out what :shrug:

Reply:
The warning also comes from other bits, but talk is mostly got with my PR

Reply:
Now that the button component was replaced by a div the warning is gone (but I nevertheless added the aria label to the div).